modules/users/data/data.yaml

```
@@ -31,15 +31,22 @@ groups:
                  'ALL = (ALL) NOPASSWD: /bin/journalctl *']
   puppet-users:
     gid: 2004
-    description: limited access on puppet111 to manage SSL certificates
-    members: [macfan]
-    privileges: ['ALL = (ALL) NOPASSWD: /root/ssl-certificate',
-                 'ALL = (ALL) NOPASSWD: /var/lib/nagios/ssl-acme']
+    description: (archived group) limited access on puppet111 to manage SSL certificates
+    members: []
+    privileges: []
+
   bastion:
     gid: 2005
     description: users who require bastion access
     members: [macfan, rhinos, universalomega, agent]
     privileges: []
+    
+  letsencrypt-admins:
+    gid: 2006
+    description: limited sudo access on puppet111 to manage SSL certificates
```

```suggestion
    description: limited sudo access to manage SSL certificates
```
Dropping puppet111, as it's one less thing to have to change if it's ever changed again. But it's up to you.